network/stream/stream.go

```diff
@@ -489,8 +468,7 @@ func (r *Registry) clientHandleOfferedHashes(ctx context.Context, p *Peer, msg *
 	// the LastIndex on the incoming message. we should seal the interval and request the subsequent
 	if lenHashes == 0 {
 		if err := p.sealWant(w); err != nil {
-			p.logger.Error("error persisting interval", "from", w.from, "to", w.to, "err", err)
-			return fmt.Errorf("error persisting interval: %w", err)
+			return fmt.Errorf("error persisting interval: from %d, to: %d, err: %w", w.from, w.to, err)
```

Comment:
This error message is not related to what the function `sealWant` suggests to do. I know that this is not the message you wrote, but it should be fixed.